index.js

```diff
@@ -166,6 +166,56 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Changes the theme based upon the value passed
+ * @param {int}     themeNum - a number passed based upon what theme button the user selected
```

Comment:
Nit: spacing after `{int}`
Perhaps the above can be rephrased to just: "based on the theme button clicked by the user" ?